Runner.php

```diff
@@ -15,32 +15,64 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
+        if(isset($this->config['include_paths'])) {
+            $this->queueWithIncludePaths();
+        } else {
+            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
+        }
+
+        $this->server->process_work(false);
+    }
+
+    public function queueWithIncludePaths() {
+        foreach ($this->config['include_paths'] as $f) {
+            if ($f !== '.' and $f !== '..') {
+
+                if (is_dir("/code$f")) {
```

Comment:
I didn't think we wrote `include_paths` with leading `/` chars. I notice you do put a `/` before `$f` on line 39. I'm guessing this has been tested, but just checking it's intentional.